controllers/bookController.js

```diff
@@ -65,7 +65,7 @@ exports.book_detail = function(req, res, next) {
     }, function(err, results) {
         if (err) { return next(err); }
         if (results.book==null) { // No results.
-            var err = new Error('Book not found');
+            let err = new Error('Book not found');
```

Comment:
Should this be a `const`? I mean, I don't think that err is reassigned after this point?